.github/workflows/release-manual.yaml

```diff
@@ -0,0 +1,45 @@
+name: Release - Manual
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag'
+        required: true
+        default: 'latest'
+        type: choice
+        options:
+          - latest
+          - nightly
+          - experimental
+
+concurrency:
+  group: changeset-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+env:
+  PNPM_VERSION: '8.15.7'
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  release:
+    name: Release
+    runs-on: shopify-ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
~Is it possible to set a branch for this? It might make testing this out a bit easier.~ ignore this, I think you choose the branch when you trigger the workflow